.github/scripts/generate-pr-plugin.sh

```diff
@@ -246,6 +250,57 @@ fi
 
 echo "Step 2: Update will now proceed with installation of new PR files..."
 echo ""
+
+# The update continues by running the install method which will extract new files
+]]>
+</INLINE>
+</FILE>
+
+<!-- Add a banner to warn user this plugin is installed -->
+<FILE Name="/usr/local/emhttp/plugins/webgui-pr-PR_PLACEHOLDER/Banner-PR_PLACEHOLDER.page" Method="install update">
+<INLINE>
+<![CDATA[
+Menu='Buttons'
+Link='nav-user'
+---
+<script>
+  $(function() {
+    // Check for updates (non-dismissible)
+    caPluginUpdateCheck("webgui-pr-PR_PLACEHOLDER.plg", {noDismiss: true});
+
+    // Create banner with uninstall link (dismissible, but returns on update)
+    var bannerMessage = "<i class='fa fa-warning' style='float:initial;'></i> " +
+                       "Modified GUI installed via <b>webgui-pr-PR_PLACEHOLDER</b> plugin. " +
+                       "<a onclick='uninstallPRPlugin()' style='cursor: pointer; text-decoration: underline;'>Click here to uninstall</a>";
+
+    // false = warning style, false = dismissible
+    addBannerWarning(bannerMessage, false, false);
+
+    // Define uninstall function
+    window.uninstallPRPlugin = function() {
+      swal({
+        title: "Uninstall PR Test Plugin?",
+        text: "This will restore all original files and remove the test plugin.",
+        type: "warning",
+        showCancelButton: true,
+        confirmButtonText: "Yes, uninstall",
+        cancelButtonText: "Cancel",
+        closeOnConfirm: false,
+        showLoaderOnConfirm: true
+      }, function(isConfirm) {
+        if (isConfirm) {
+          // Execute plugin removal using openPlugin (Unraid's standard method)
+          openPlugin("plugin remove webgui-pr-PR_PLACEHOLDER.plg", "Removing PR Test Plugin");
```

Comment:
Nitpick:  Code assuming that the PR gets uninstalled within 3 seconds.

Removing the setTimeout and changing to
` openPlugin("plugin remove webgui-pr-PR_PLACEHOLDER.plg", "Removing PR Test Plugin","","refresh");`

Should automatically reload the page when the uninstall is completed